packages/wrangler/src/init.ts

```diff
@@ -620,3 +688,191 @@ async function findPath(
 		});
 	}
 }
+
+async function getWorkerConfig(
+	accountId: string,
+	fromDashScriptName: string,
+	{
+		defaultEnvironment,
+		environments,
+	}: {
+		defaultEnvironment: string;
+		environments: ServiceMetaDataRes["environments"];
+	}
```

Comment:
This seems worth extracting out into an extra type (as well as extracting the ServiceMetaDataRes["environments"] type into a separate one)

Reply:
I prefer not to. It is not going to be reused and its functionally the same either way.